reana_client/cli/analyses.py

```diff
@@ -30,6 +30,28 @@
 from ..utils import load_reana_spec, load_workflow_spec
 
 
+@click.command()
+@click.option('-f',
+              '--file',
+              type=click.Path(exists=True, resolve_path=True),
+              default=reana_yaml_default_file_path,
+              help='REANA specifications file describing the workflow and '
+                   'context which REANA should execute.')
+@click.pass_context
+def validate(ctx, file):
```

Comment:
There is only left to add this to `reana_client/cli/__init__.py`:
```python
cli.add_command(analyses.validate)
```
so click takes it into account

Reply:
:man_facepalming: 
Fixed

Reply:
thanks!! we :shipit: